src/Canvas.cc

```diff
@@ -200,7 +200,7 @@ int
 Canvas::EIO_AfterToBuffer(eio_req *req) {
 #endif
 
-  NanScope();
+  Nan::HandleScope scope;
```

Comment:
don't remove any of these that aren't in the `NAN_METHOD` or `NAN_SETTER` functions, they aren't implicit so these are needed to avoid leaks